projects/plugins/jetpack/modules/subscriptions/subscribe-modal/subscribe-modal.css

```diff
@@ -52,6 +52,7 @@
 
 .jetpack-subscribe-modal__modal-content h2 {
     margin: 4px;
+	text-align: center;
```

Comment:
Could we apply this to the block markup itself?

Otherwise we break the editing experience from modal to the template.

The same is true for the margin; let's instead use group block with padding values.

Reply:
I've moved this into the block markup.